recipes/pygal/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pygal" %}
+{% set version = "2.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9204f05380b02a8a32f9bf99d310b51aa2a932cba5b369f7a4dc3705f0a4ce83
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - lxml
+    - cairosvg
+    - tinycss
+    - cssselect
```

Comment:
It seems like something is missing (according to the CI). Maybe `pyquery` and `flask`?

Reply:
Apparently it's complaining about `setuptools` and asking to add all dependencies to the `meta.yaml` file, but this package doesn't have any required dependencies, only some optional ones that I've added.

Reply:
OK, I think I got all dependencies now. But the tests are failing: `ModuleNotFoundError: No module named 'pkg_resources'`

Reply:
@Matt-BF, for most python packages all you typically need to include in `requirements/host` is python, pip, and setuptools, the _runtime_ dependencies should be listed in `requirements/run`, but it looks here like you have nothing listed at all (except python).

Specifically the builds are failling because your package has a top-level import of `pkg_resources` which is provided by `setuptools`, so that definitely needs to be included in `requirements/run`.

Reply:
@duncanmmacleod yeah, that's what I thought, but for some reason this package freaks out if the dependencies go to run instead of host (something to do with `egg_info`), not sure what I'm doing wrong. Also, adding `setuptools` to `requirements/host` didn't solve the `ModuleNotFoundError`. Maybe I should try adding `setuptools` to `requirements/run`?

Reply:
Because your package (as opposed to the build) imports `pkg_resources`, you _definitely_ need `setuptools` in `requirements/run`. The build (`host`) and run/test (`run`) environments are completely independent, so including a package in one list has no bearing on the other (typically)

Reply:
@duncanmmacleod That worked perfectly, thank you very much for all your help! Only issue now is the linter isn't happy about the way I wrote `skip: True # [py2k]`